release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -33,7 +33,7 @@ Starting with {product-title} {product-version}, Extended Update Support (EUS) i
 {product-title} {product-version} is an Extended Update Support (EUS) release. More information on Red Hat OpenShift EUS is available in link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[OpenShift Life Cycle] and link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
 
 //TODO: The line below should be used when it is next appropriate. Revisit in August 2023 time frame.
-Maintenance support ends for version 4.12 on 25 January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Maintenance support ends for version 4.12 on 17 January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```

Comment:
From https://access.redhat.com/support/policy/updates/openshift,
Maintenance support ends for version 4.12 on July 17, 2024, Extended update support ends on January 17, 2025

Reply:
@liangxia How does it look now?